uiowa/uiowa
# Feature: blockquote preview fixes for layout edit screen

Noticed the blockquote styles were being overridden by layout builder when editing a blockquote block and this pr updates the preview styles.

<img width="1653" alt="Screen Shot 2021-11-18 at 10 45 26 AM" src="https://user-images.githubusercontent.com/1036433/142459192-2275e634-e7c5-43ab-89da-eb7ae2353294.png">

# How to test

`yarn workspace uids_base build`
- Test the Left and Right variants of https://sandbox.local.drupal.uiowa.edu/blockquotes without an image and verify that you can see the left and right gold rule when editing a blockquote block in layout builder. 
- Test all other variants and verify that you can see the correct preview of the blockquote after saving the block and then editing the block again.
